### PR TITLE
Update dependencies, GitHub actions, vitest

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

**Comment by jobo322** [comment]
@targos could you have a look, please

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "eslint-fix": "npm run eslint -- --fix",
     "prepublishOnly": "npm run compile",
     "test": "npm run test-coverage && npm run eslint",
-    "test-only": "jest",
-    "test-coverage": "jest --coverage"
+    "test-only": "vitest --globals",
```
Comment (targos) [inline package.json:17]: Please don't use `--globals`.

**Comment by jobo322** [comment]
> Please follow the guide in [mljs/ml#174](https://github.com/mljs/ml/issues/174). There are some things missing.

thanks

**Comment by jobo322** [comment]
@targos seems to me the cleanup is done

**Comment by targos** [comment]
Did you check that the rollup script works?

**Comment by targos** [comment]
I added you as a collaborator. Can you push your branch to this repo and open a new PR so the GitHub actions can run?

**Comment by jobo322** [comment]
> Did you check that the rollup script works?

sorry I forgot to change the extension

**Pull request closed without merging**